pkg/cluster/types.go

```diff
@@ -20,6 +21,7 @@ const (
 	Replica PostgresRole = "replica"
 )
 
+// PodEventType describes value of event type
```

Comment:
It sounds strange, how data type can describe a value? Maybe 'PodEventType is a data type that represents type of a pod-related event'?